src/toolkit/timeseries.cc

```diff
@@ -8,6 +8,15 @@ void RecordTimeSeries<POWER>(cyclus::Agent* agent, double value) {
   RecordTimeSeries<double>("Power", agent, value);
 }
 
+template <>
+void RecordTimeSeries<ENRICH_SWU>(cyclus::Agent* agent, double value) {
```

Comment:
@rwcarlsen, do you have a case for naming this `ENRICH_SWU` vs. `SWU`? I know it gives semantic meaning, but in general we have eschewed semantics in favor of explicit joining in order to discover agent/prototype type.